### PR TITLE
feat: replaced Python's request with curl command to reproduce error

### PR DESCRIPTION
**Pull request by presto21**
More convenient if the error could be reproduced with a curl command.

Changes: 

- removed python's request example
- replaced it with a curl command

### CODE REVIEW

File: src/ibm_service_validator/cli/handlers/output_handler.py [hunk 1]
```diff
@@ -160,26 +166,52 @@ def display_single_test(
...
+) -> None:
     for idx, check in enumerate(checks, 1):
         message: Optional[str]
         if check.message:
             message = f"{idx}. {check.message}"
         else:
             message = None
         example = cast(SerializedCase, check.example)
-        display_example(example, color, check.name, message, seed)
+        display_example(example, color, method, path, check.name, message, seed)
```
Comment (barrett-schonefeld) [inline src/ibm_service_validator/cli/handlers/output_handler.py:182]: Aren't the method and path included in the SerializedCase, `example`? May not need to pass these as arguments.

File: src/ibm_service_validator/cli/handlers/output_handler.py [hunk 2]
```diff
@@ -160,26 +166,52 @@ def display_single_test(
...
         # Display every time except the last check
         if idx != len(checks):
             click.echo("\n")
 
 
+def produce_curl_command(
+    case: SerializedCase, path: str, method: str, request_example: str
```
Comment (barrett-schonefeld) [inline src/ibm_service_validator/cli/handlers/output_handler.py:189]: Reiterating what I said above. The `case` should have the `path` and `method` info.

File: src/ibm_service_validator/cli/handlers/output_handler.py [hunk 3]
```diff
@@ -160,26 +166,52 @@ def display_single_test(
...
+
+    return curl.format(method=method, headers=headers, data=data, base_url=base_url)
+
+
+# pylint: disable=too-many-arguments
 def display_example(
     case: SerializedCase,
     color: str,
+    method: str,
+    path: str,
```
Comment (barrett-schonefeld) [inline src/ibm_service_validator/cli/handlers/output_handler.py:214]: Reiterating what I said above. The case should have the path and method info.

**Review by barrett-schonefeld — COMMENTED** [review]
Would like to see some tests that ensure the curl command is correct and can be used to call an API.

**Comment by rmkeezer** [comment]
PR moved to https://github.com/IBM/service-validator/pull/12

**Pull request closed without merging**